uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/modules/shared-settings.spec.js

```diff
@@ -35,7 +35,7 @@ define([
       });
 
       it('contains the list of allowed mime types', function () {
-        expect(sharedSettings.fileUploader.allowedMimeTypes).toBeDefined();
+        expect(sharedSettings.fileUploader.mimeTypesMap).toBeDefined();
```

Comment:
If I want to compare it with the exact object I have to manually copy paste the whole object, which I dont think will bring any benefit because in the end it just a static object and I think its enough to keep it in the settings file only.